app/controllers/supporting_evidences_controller.rb

```diff
@@ -4,7 +4,20 @@ def show
     claim_summary = BaseViewModel.build(:claim_summary, claim)
     supporting_evidence = BaseViewModel.build(:supporting_evidence, claim, 'supporting_evidences')
 
+    generate_urls supporting_evidence
+
     @pagy, @supporting_evidence = pagy_array(supporting_evidence)
     render locals: { claim:, claim_summary: }
   end
+
+  private
+
+  def generate_urls(supporting_evidence)
+    supporting_evidence.each do |item|
+      item.download_url = S3_BUCKET
+                          .object(item.file_path)
+                          .presigned_url(:get, expires_in: 900,
```

Comment:
to confirm this is 1.5min or 1.5 hours - if the first it doesn;t seem long enough

would also prefer this to be an const. :)

Reply:
This is 15 mins (900 seconds) - happy to set this as a const and add a comment